recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 source:
   fn: conda-execute-{{ version }}.tar.gz
   url: https://github.com/conda-tools/conda-execute/archive/v{{ version }}.tar.gz
-  sha256: c3182c0736fbfed07bc7fdd65e0331e62a440a3158b56cd8d0c5f249fd780489
+  sha256: 21388f4fd0e4e604b95231f207481ece08ba25cf76c6af63a94dcb63716c3c47
 
 build:
-  number: 2
+  number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Thoughts on making this a [`noarch: python`]( http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages ) package?